### ISSUE TITLE
Feature Suggestion: Optional beep on various events

### ISSUE THREAD

**Issue by eric116** [Hoektronics/BotQueue#139]
Option for printer to beep (M300) on specified events, such as print completion, or possibly certain errors. Would probably be a feature for users with a smaller number of bots. Would somehow need to send that gcode while the printer is in working mode.

**Comment by Jnesselr** [comment]
For print completion, you could just make that the last gcode in your file.
The main problem with doing it for errors is that the main errors it
detects are being unable to talk to the bot. :D

**Comment by eric116** [comment]
True. And yeah, unfortunately most errors are things like extruder jams and poor bead height that only humans can see.
